dist/AppImageBuilder.yml

```
@@ -0,0 +1,49 @@
+version: 1
+
+script:
+- |
+  icon_path="{{APPIMAGE_PATH}}/usr/share/icons/hicolor/128x128/apps"
+  mkdir -p "${icon_path}"
+  cp -a dist/evdevhook2.png "${icon_path}"
+
+AppDir:
+  path: "{{APPIMAGE_PATH}}"
+
+  app_info:
+    id: org.v1993.evdevhook2
+    name: evdevhook2
+    version: "{{APPIMAGE_VERSION}}"
+    icon: evdevhook2
+    exec: usr/bin/evdevhook2
+    exec_args: $@
+
+  apt:
+    arch:
+      - "{{APPIMAGE_APT_ARCH}}"
+    sources:
+      - sourceline: deb [arch=amd64,i386] http://archive.ubuntu.com/ubuntu/ "{{APPIMAGE_APT_DISTRO}}" main
+        key_url: "http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x{{APPIMAGE_APT_PUBKEY}}"
+      - sourceline: deb [arch=amd64,i386] http://archive.ubuntu.com/ubuntu/ "{{APPIMAGE_APT_DISTRO}}"-updates main
+      - sourceline: deb [arch=amd64,i386] http://security.ubuntu.com/ubuntu/ "{{APPIMAGE_APT_DISTRO}}"-security main
+      - sourceline: deb [arch=arm64,armhf] http://ports.ubuntu.com/ubuntu-ports/ "{{APPIMAGE_APT_DISTRO}}" main
+      - sourceline: deb [arch=arm64,armhf] http://ports.ubuntu.com/ubuntu-ports/ "{{APPIMAGE_APT_DISTRO}}"-updates main
+      - sourceline: deb [arch=arm64,armhf] http://ports.ubuntu.com/ubuntu-ports/ "{{APPIMAGE_APT_DISTRO}}"-security main
```

One final thing before merge:

```suggestion
      - sourceline: deb [arch=amd64,arm64] http://archive.ubuntu.com/ubuntu/ "{{APPIMAGE_APT_DISTRO}}" main
        key_url: "http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x{{APPIMAGE_APT_PUBKEY}}"
      - sourceline: deb [arch=amd64,arm64] http://archive.ubuntu.com/ubuntu/ "{{APPIMAGE_APT_DISTRO}}"-updates main
      - sourceline: deb [arch=amd64,arm64] http://security.ubuntu.com/ubuntu/ "{{APPIMAGE_APT_DISTRO}}"-security main
```

This should work, I think? `i386` and `armhf` go unused anyways.

---

No, I don't think so, because `archive.ubuntu.com` has only `amd64` and `i386` but no `arm64`. Anything else is in `ports.ubuntu.com`. You can check that using a browser.

---

Ah, I've missed that part, thanks. Then probably just remove `i386` and `armhf` from the source list.

---

Done.